vignettes/zip_zap_models.Rmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Devel: ZIP and ZAP models"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Devel: ZIP and ZAP models}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+header-includes:
+  - \newcommand{\bm}[1]{\boldsymbol{#1}}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE
+)
+```
+
+(Vignette under construction!)
+
+```{r setup, eval=TRUE, echo=TRUE, include=TRUE}
+library(tidyverse)
+library(inlabru)
+library(terra)
+library(sf)
+library(RColorBrewer)
+```
+
+# Count model
+
+In addition to point process models, `inlabru` can handle models with positive integer response, e.g. abundance models where species counts are recorded in every observed location. Count models can be seen as coarse aggregations of point process models.
+
+The following example uses `gorillas` dataset. In order to obtain the counts data we rasterize the species count to the same granularity as the spatial covariates available for the `gorilla` data, and then aggregate the pixels to cover 25 times bigger area (5x5 pixels in original covariate raster dimensions). Finally, we mask the regions outside of study area.
+
+```{r fig-count-raster, fig.cap="Counts of gorilla nests", fig.width=7, fig.height=5,out.width="80%", fig.align='center'}
+data(gorillas_sf)
```

Comment:
Following R guidelines, the data is now available via the lazydata mechanism, so no need to do `data(gorillas_sf)` when inlabru is loaded. A `data()` call is only needed to disambiguate between data from different packages, and even then one can do `inlabru::gorillas_sf` instead.